hwangJeongSoo/db/migrations/20221124072109_create_users_table.sql

```diff
@@ -0,0 +1,16 @@
+-- migrate:up
+CREATE TABLE
+    users (
+        id INT NOT NULL AUTO_INCREMENT,
+        name VARCHAR(50) NOT NULL,
+        email VARCHAR(200) NOT NULL,
+        profile_image VARCHAR(1000) NULL,
+        password VARCHAR(200) NOT NULL,
+        created_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+        updated_at TIMESTAMP NULL ON UPDATE CURRENT_TIMESTAMP,
+        PRIMARY KEY (id),
+        UNIQUE (email)
```

Comment:
```suggestion
        CONSTRAINT users_email_ukey UNIQUE (email)
```

- `CASE` 이미 unique 제약 조건을 작성하였으나, CONSTRAINT 키워드를 활용하지 않은 경우
    - 제약에는 이름을 붙일 수 있습니다. 제약에 이름을 붙이면 나중에 관리하기 쉬워지므로 가능한 한 이름을 붙이도록 해주세요! 제약 이름은 CONSTRAINT 키워드를 이용해서 다음과 같이 지정합니다.
    
    ```sql
    CONSTRAINT users_email_ukey UNIQUE (email)
    
    # ex 외래키 ) CONSTRAINT posts_user_id_fkey FOREIGN KEY (user_id) REFERENCES users(id)
    
    # description) CONSTRAINT 테이블명_컬럼명_키타입 [제약조건] (컬럼명) 
    ```